cmd/kinflate/README.md

```diff
@@ -13,6 +13,8 @@ go get k8s.io/kubectl/cmd/kinflate
 
 After running the above command, `kinflate` should get installed in your `GOPATH/bin` directory.
 
+Note: `kinflate` requires 1.10 or newer
```

Comment:
Assume 1.10 is available. When running `kinflate inflate -f . | kubectl apply -f -`, kubectl is also using 1.10. Is that right?

Reply:
kubectl doesn't have to use golang 1.10. It's just a binary, which can be downloaded or compiled. 
Using golang 1.10 to compile should work fine, it should be backward compatible.